scripts/tasks/storybook.ts

```diff
@@ -49,9 +49,10 @@ function getCommonOptions(): StorybookCommonOptions {
   console.log(`node heap limit = ${require('v8').getHeapStatistics().heap_size_limit / (1024 * 1024)} MB`);
 
   const localConfigDir = path.join(process.cwd(), '.storybook');
+  const localStaticDir = path.join(process.cwd(), 'static');
 
   return {
-    staticDir: [path.join(process.cwd(), 'static')],
+    staticDir: fs.existsSync(localStaticDir) ? [localStaticDir] : [],
```

Comment:
why this change?

Reply:
When running the `storybook:build` command - error below was being thrown because static folder does not exist.

![image](https://user-images.githubusercontent.com/8649804/123680013-0ecdd400-d7fd-11eb-9e01-94ced1ee2cf6.png)